### PR TITLE
fix Russian material translations

### PR DESCRIPTION
**Pull request by yjbanov**

### CODE REVIEW

File: packages/flutter/lib/src/material/i18n/material_ru.arb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   "licensesPageTitle": "Лицензии",
-  "cancelButtonLabel": "ОТМЕНА",
+  "cancelButtonLabel": "ОТМЕНИТЬ",
   "closeButtonLabel": "ЗАКРЫТЬ",
-  "continueButtonLabel": "ПРОДОЛЖАТЬ",
-  "copyButtonLabel": "копия",
-  "cutButtonLabel": "порез",
+  "continueButtonLabel": "ПРОДОЛЖИТЬ",
+  "copyButtonLabel": "Скопировать",
+  "cutButtonLabel": "Вырезать",
```
Comment (Hixie) [inline packages/flutter/lib/src/material/i18n/material_ru.arb]: button labels should be all uppercase

Comment (yjbanov) [reply packages/flutter/lib/src/material/i18n/material_ru.arb]: Done.

**Comment by yjbanov** [comment]
Clearly I don't know how to rebase. Stay tuned.

**Comment by yjbanov** [comment]
Dunno how @B3rn475 ended up in the commit metadata, but the diff is correct now. PTAL.

**Review by HansMuller — COMMENTED** [review]
You'll need to regenerate localizations.dart too

**Comment by yjbanov** [comment]
> You'll need to regenerate localizations.dart too

Done.

**Comment by HansMuller** [comment]
LGTM. Thanks for making sure that these translations are correct!

**Pull request merged**